charts/redpanda/values.go

```diff
@@ -1602,8 +1611,16 @@ func (l *SchemaRegistryListeners) TrustStores(tls *TLS) []*TrustStore {
 	return tss
 }
 
-func (sr *SchemaRegistryListeners) ConsoleTLS(tls *TLS) config.SchemaTLS {
-	t := config.SchemaTLS{Enabled: sr.TLS.IsEnabled(tls)}
+type SchemaTLS struct {
```

Comment:
Could we just use a single struct for this? It looks like all the keys are the same. Please add comments/notes about why this struct is defined as well.

Reply:
I will move to single struct.